recipe/meta.yaml

```diff
@@ -1,42 +1,64 @@
+{% set sip_version = "13.10.0" %}
+{% set pyqt_builder_version = "1.15.4" %}
+{% set webengine_name = "PyQtWebEngine" %}
+{% set webengine_version = "6.8.1" %}
+{% set charts_name = "PyQt6-Charts" %}
+{% set charts_version = "6.8.0" %}
+{% set PYTHON = PYTHON | default("$PYTHON") %}
+{% set python_min = python_min | default("3.6") %}
 
 
 package:
-  name: pyqt-split
+  name: pyqt6-split
   version: {{ version }}
 
 source:
-  - url: https://pypi.io/packages/source/P/PyQt{{ version[0] }}/PyQt{{ version[0] }}-{{ version }}.tar.gz
-    sha256: dc41e8401a90dc3e2b692b411bd5492ab559ae27a27424eed4bd3915564ec4c0
+  - url: https://pypi.org/packages/source/{{ pyqt_builder_name[0] }}/{{ pyqt_builder_name }}/{{ pyqt_builder_name }}-{{ pyqt_builder_version }}.tar.gz
+    sha256: 39f8c75db17d9ce17cb6bbf3df1650b5cebc1ea4e5bd73843d21cc96612b2ae1
+    folder: pyqt-builder
+    patches:
+      #- patches/0001-Use-conda-sysroot-when-building-recipes.patch
+      - patches/0002-disable-test-execution-cross.patch
+      - patches/0003-find-sip-distinfo.patch
+      - patches/0004-Use-conda-sysroot-when-building-recipes.patch
+
+  - url: https://pypi.org/packages/source/{{ pyqt_name[0] }}/{{ pyqt_name }}/{{ pyqt_name.replace('-', '_').lower() }}-{{ version }}.tar.gz
+    sha256: 91d937d6166274fafd70f4dee11a8da6dbfdb0da53de05f5d62361ddf775e256
     folder: pyqt
 
-  - url: https://pypi.io/packages/source/P/PyQt5-sip/PyQt5_sip-{{ sip_version }}.tar.gz
-    sha256: 10d9bfa9f59f0fd1cad81be187479316ffc95684f573efea94512cb4257d2b17
+  - url: https://pypi.org/packages/source/{{ pyqt_sip_name[0] }}/{{ pyqt_sip_name }}/{{ pyqt_sip_name.replace('-', '_').lower() }}-{{ sip_version }}.tar.gz
+    sha256: d6daa95a0bd315d9ec523b549e0ce97455f61ded65d5eafecd83ed2aa4ae5350
     folder: pyqt_sip
 
-  - url: https://pypi.io/packages/source/P/PyQtWebEngine/PyQtWebEngine-{{ webengine_version }}.tar.gz
-    sha256: cedc28f54165f4b8067652145aec7f732a17eadf6736835852868cf76119cc19
-    folder: pyqt_webengine
+  # - url: https://pypi.org/packages/source/{{ webengine_name[0] }}/{{ webengine_name }}/{{ webengine_name }}-{{ webengine_version }}.tar.gz
+  #   sha256: cedc28f54165f4b8067652145aec7f732a17eadf6736835852868cf76119cc19
+  #   folder: pyqt_webengine
 
-  - url: https://pypi.io/packages/source/P/PyQtChart/PyQtChart-{{ charts_version }}.tar.gz
-    sha256: e2cd55a8a72cef99bc0126f3b1daa914eb5f21e20a70127b6985299f1dc50107
+  - url: https://pypi.org/packages/source/{{ charts_name[0] }}/{{ charts_name }}/{{ charts_name.replace('-', '_') }}-{{ charts_version }}.tar.gz
+    sha256: f86705b8740e3041667ce211aeaa205b750eb6baf4c908f4e3f6dc8c720d10f1
     folder: pyqt_charts
 
 build:
-  number: 5
+  number: 0
 
 outputs:
-  - name: pyqt5-sip
-    version: {{ sip_version }}
-    script: build-pyqt-sip.sh  # [not win]
-    script: bld-pyqt-sip.bat  # [win]
+  # Not actually exporting this package, just experimenting to see if
+  # this needs to be patched.
+  - name: my-pyqt6-builder
```

Comment:
Any outputs here will be published. Once you're done debugging please remove this.

Reply:
Will they be published even if it's in a PR? I would have thought it would have only happened if it was merged into `main`. My plan was to remove it once everything was green and applied the changes upstream to `pyqt-builder-feestock`.

Reply:
No, during a PR is fine.